models.py

```diff
@@ -0,0 +1,72 @@
+from typing import List, Union
```

Comment:
Name of this file is too generic, what about something like `jiramodels` or something like that.